wiki/Tournaments/TWC/2021/en.md

```diff
@@ -0,0 +1,220 @@
+---
+tags:
+  - TWC
+  - TWC2021
+---
+
+# osu!taiko World Cup 2021
+
+![TWC 2021 logo](/wiki/shared/news/banners/TWC_2021.jpg)
+
+The **osu!taiko World Cup 2021** (***TWC 2021***) is a country-based osu!taiko tournament hosted by the [osu! team](/wiki/People/The_Team). It is the eleventh installment of the osu!taiko World Cup.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2021-02-04/2021-02-18 |
+| Live drawings | 2021-02-27 (14:00 UTC) |
+| Group Stage | 2021-03-06/2021-03-07 |
+| Round of 16 | 2021-03-13/2021-03-14 |
+| Quarterfinals | 2021-03-20/2021-03-21 |
+| Semifinals | 2021-03-27/2021-03-28 |
+| Finals week 1 | 2021-04-03/2021-04-04 |
+| Finals week 2 | 2021-04-10/2021-04-11 |
+
+## Prizes
+
+| Placing | Prizes |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | $150 per team member, unique profile badge, *osu!taiko Champion* user title for one year |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | $80 per team member, unique profile badge |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | $40 per team member, unique profile badge |
+
+## Organisation
+
+The osu!taiko World Cup 2021 is run by various community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Manager | ![][flag_AR] [juankristal](https://osu.ppy.sh/users/443656), ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416) |
+| Mappool selector | ![][flag_US] [Backfire](https://osu.ppy.sh/users/263110), ![][flag_US] [incandescence](https://osu.ppy.sh/users/6256027), ![][flag_AU] [Jaye](https://osu.ppy.sh/users/4841352), ![][flag_JP] [_Rise](https://osu.ppy.sh/users/5217107) |
+| Commentator | *-- To be announced --* |
+| Referee | ![][flag_ES] [Deif](https://osu.ppy.sh/users/318565), ![][flag_AU] [ill onion](https://osu.ppy.sh/users/8306102), ![][flag_BR] [LeoFLT](https://osu.ppy.sh/users/3668779), ![][flag_NL] [nik](https://osu.ppy.sh/users/10077264), ![][flag_DE] [p3n](https://osu.ppy.sh/users/123703), ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811), ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416), ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) |
+| Statistician | ![][flag_BR] [LeoFLT](https://osu.ppy.sh/users/3668779), ![][flag_DE] [Nwolf](https://osu.ppy.sh/users/1910766) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1241159)
+- [Livestream](https://www.twitch.tv/osulive)
+- **[Registration Page](https://osu.ppy.sh/community/tournaments/27)**
+
+## Ruleset
+
+### Tournament rules
+
+1. The osu!taiko World Cup is a country-based team tournament, played on the osu!taiko game mode.
+   - **While this competition is planned as a 2 versus 2 setup, this might change depending on the amount of incoming registrations.**
```

Comment:
```suggestion
   - **While this competition is planned as a 2 versus 2 setup, this might change depending on the number of incoming registrations.**
```